<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaInicio</class>
 <widget class="QMainWindow" name="VentanaInicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 53)</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(86, 255, 237)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <widget class="QPushButton" name="boton_iniciar_partida">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 248, 91)</string>
      </property>
      <property name="text">
       <string>Iniciar Partida</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_ventana_ranking">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 248, 91)</string>
      </property>
      <property name="text">
       <string>Ventana Ranking</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linea_texto_usuario">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Ingrese su usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>281</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
